modules/planning/common/frame.cc

```diff
@@ -136,12 +136,25 @@ bool Frame::InitReferenceLineInfo() {
     ++segments_iter;
   }
 
+  uint32_t info_size = reference_line_info_.size();
+  DCHECK_LE(info_size, 2);
+
   for (auto &info : reference_line_info_) {
     if (!info.Init()) {
       AERROR << "Failed to init reference line";
       return false;
     }
   }
+
+  // set lane change path flag in reference line info
+  for_each(
+      reference_line_info_.begin(), reference_line_info_.end(),
+      [info_size](ReferenceLineInfo &info) {
+        if ((info_size == 2) && info.Lanes().NextAction() == routing::FORWARD) {
```

Comment:
This may not true. when there are three parallel lanes, you are on the right most lane, and want to change to middle lane then change to left most lane, the middle lane's NextAction is LEFT